healthcare/v1/src/main/java/snippets/healthcare/fhir/resources/FhirResourceSearchGet.java

```diff
@@ -22,42 +22,63 @@
 import com.google.api.client.json.JsonFactory;
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.healthcare.v1.CloudHealthcare;
-import com.google.api.services.healthcare.v1.CloudHealthcare.Projects.Locations.Datasets.FhirStores.Fhir.Search;
 import com.google.api.services.healthcare.v1.CloudHealthcareScopes;
-import com.google.api.services.healthcare.v1.model.HttpBody;
-import com.google.api.services.healthcare.v1.model.SearchResourcesRequest;
 import com.google.auth.http.HttpCredentialsAdapter;
 import com.google.auth.oauth2.GoogleCredentials;
 import java.io.IOException;
 import java.net.URISyntaxException;
 import java.util.Collections;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.client.methods.RequestBuilder;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClients;
 
 public class FhirResourceSearchGet {
   private static final String FHIR_NAME =
       "projects/%s/locations/%s/datasets/%s/fhirStores/%s/fhir/%s";
   private static final JsonFactory JSON_FACTORY = new JacksonFactory();
   private static final NetHttpTransport HTTP_TRANSPORT = new NetHttpTransport();
 
-  public static void fhirResourceSearchGet(String fhirStoreName, String resourceType)
+  public static void fhirResourceSearchGet(String resourceName)
       throws IOException, URISyntaxException {
-    // String fhirStoreName =
+    // String resourceName =
     //    String.format(
     //        FHIR_NAME, "project-id", "region-id", "dataset-id", "fhir-store-id");
     // String resourceType = "Patient";
 
-    // Initialize the client, which will be used to interact with the service.
-    CloudHealthcare client = createClient();
+    // The endpoint URL for the Healthcare API. Required for HttpClient.
+    String rootUrl = "https://healthcare.googleapis.com";
```

Comment:
I think this should be a final variable at the top

Reply:
Done.